pom.xml

```diff
@@ -301,6 +301,7 @@
           </dependency>
         </dependencies>
         <configuration>
+          <cleanupDaemonThreads>false</cleanupDaemonThreads>
```

Comment:
why don't we want these cleaned up?

Reply:
Because the okhttp daemon threads take ~ 15 seconds to cleanup. With this set to false maven completes and the daemon threads are cleaned up faster.